public/js/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-/* global jQuery io RQ_PARAMS KeyControl ServoConfig */
```

Comment:
After being able to review the code side-by-side, I noticed even after doing a git fetch and pull, not all of the servo changes made it to my local repository. Luckily it looks fairly simple to merge, but sorry about that! I will be more careful next time.